pepper-apis/dss-database/src/main/resources/db-changes/tenant-migration.xml

```diff
@@ -3,12 +3,18 @@
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+        https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
-    <changeSet id="DDP-2606-backfill-auth0-tenant" author="andrew">
+    <changeSet id="DROP_AUTH0_DOMAIN_COLUMN" author="Dmitrii">
+        <preConditions onError="MARK_RAN" onFail="MARK_RAN">
```

Comment:
Same here. But the difference is that this tenant-migration changeset is executed from the Java code and under some conditions. So we just trying to remove the column in case it exists.

Reply:
OK. I tested this PR by:
1. Creating a  brand new database/nothing at all in it.
2. Run backend server with code currently in `develop` branch (NOT THIS PR's code) with configuration that executes liquibase. server started and created the database.
3. Manually/directly in the database I added the auth0_domain column to client.
4. Rebuilt backend server with the code from this P.R.
5. Ran backend server with configuration that allows it to execute liquibase.

Got the following error during backend server startup (This is the sort of error I expect whenever a previously applied liquibase xml file is modified).
`
08:09:15.565   C: S: [main] ERROR o.b.ddp.DataDonationPlatform Could not start ddp
org.broadinstitute.ddp.exception.DDPException: Error running liquibase migrations
	at org.broadinstitute.ddp.util.LiquibaseUtil.runLiquibase(LiquibaseUtil.java:75)
	at org.broadinstitute.ddp.DataDonationPlatform.start(DataDonationPlatform.java:296)
	at org.broadinstitute.ddp.DataDonationPlatform.main(DataDonationPlatform.java:257)
Caused by: liquibase.exception.ValidationFailedException: Validation Failed:
     1 change sets check sum
          db-changes/tenant-migration.xml::DDP-2606-backfill-auth0-tenant::andrew was: 8:eeaec3c77a1e396cd33342c067328662 but is now: 8:6e40de2ad6365af9706da4452d68378e

	at liquibase.changelog.DatabaseChangeLog.validate(DatabaseChangeLog.java:276)
	at liquibase.Liquibase.rollback(Liquibase.java:701)
	at liquibase.Liquibase.rollback(Liquibase.java:686)
	at liquibase.Liquibase.rollback(Liquibase.java:673)
	at org.broadinstitute.ddp.util.LiquibaseUtil.runMigrations(LiquibaseUtil.java:145)
	at org.broadinstitute.ddp.util.LiquibaseUtil.runPepperAPIsGlobalMigrations(LiquibaseUtil.java:106)
	at org.broadinstitute.ddp.util.LiquibaseUtil.runLiquibase(LiquibaseUtil.java:65)
	... 2 common frames omitted
`

Reply:
Ah, I see you point now. We're talking about changing existing script that affects it's hash. Yep. Got it.